### ISSUE TITLE
Extend grid southward and displace South Pole (at least 86S)

### ISSUE THREAD

**Issue by claireyung** [claireyung/mom6-panAn-iceshelf-tools#1]
•	Extend grid southward and displace South Pole (at least 86S) (Claire to generate 1/4deg (OM3-like) and a higher-res test grid that will be used to test the ice thickness and IC tools)
o	Test if sufficient speed up (larger dx) achieved by using the normal OM3 grid up to the ice shelf fronts (82S?) and then displaced pole south of that. Alternatively, start warping grid from 65S as in Northern hemisphere?
o	Ocean model grid generation tool works and has capability to do displaced pole. Probably requires some optimisation of where the pole should be displaced to.
o	Requires: decision on resolution: 1/20th or 1/24th? Test case at a lower resolution, say 1/12th? Ideally the same grid that is used for a high-resolution OM3 version to reduce repeated topo-edits workload. But the tool works for any resolution because resolution is an input variable.

**Comment by claireyung** [comment]
### Example grid for 1/20th degree model with displaced pole

Similarly to above, I created a grid for 1/20th degree resolution. It is global (we can truncate it at 35S later I guess), Mercator to 70S and warped from 75S for the displaced pole. 75S is further north than the ice shelf calving fronts. There is a sizeable discontinuity in dy at 75S, probably not ideal but hard to get around if want to both a) make a difference with the smallest grid size and b) not start the displaced pole too far north. Here we have a increase in smallest ocean grid cell size of approximately 400m to 560m. Is that even worth it?

File can be found at `/g/data/x77/cy8964/mom6/ocean_hgrid_005_dsp_v13.nc`

Plots: 

<img width="960" alt="Image" src="https://github.com/user-attachments/assets/78d8a4b1-be5d-4cfd-9590-f82e503450f5" />

<img width="946" alt="Image" src="https://github.com/user-attachments/assets/a73333f9-e64a-4221-83c6-273d7fa37785" />

**Comment by aekiss** [comment]
Cross-reference to some displaced-pole and other grid discussion: https://github.com/COSIMA/access-om3/issues/172

**Comment by anton-seaice** [comment]
> there is quite a large discontinuity in dyt at the south pole cap boundary if we significantly displace the pole. I don't know if that's an issue?

There are published grids with an offset north pole (e.g. CESM has at least one, they might be called gx1 and gx3) do they have this discontinuity in dy?

**Comment by aekiss** [comment]
Is the discontinuity unavoidable, since dy is stretched on one side of the circle and compressed on the other? I guess it depends on how the distortion is done.

**Comment by adele-morrison** [comment]
From discussion at 7th Feb meeting: we won't know if the small gridcells are a problem until we run the model (maybe the flow/wave speed there is small anyway). So we decided to shelve shifting the pole until we find if we need it or not. For now proceed with a standard un-displaced pole.

Though if someone (e.g. @ezhilsabareesh8 ?) wants to have a play and finds we could get a factor of say 2 improvement in grid cell size without too large discontinuities in dy that could be a good extra check.

**Comment by aekiss** [comment]
Agreed we shouldn't try to fix this until we have established it's a problem. In case we get back to it, here are some thoughts:

Below is a plot of the mesh (every 16th cell I believe) from the first of [@claireyung's slides](https://drive.google.com/file/d/1XBhd43NJMO1O21Xrsq0597OL2CCL0qvC/view?usp=sharing), on which I've marked the join to the displaced grid as a red circle. There's a discontinuity in dy at the join in the Ross and Weddell (cyan boxes) and also an aspect ratio (dx/dy) which is far from square. There are some longitudes where there is no discontinuity in dy and a square aspect ratio (green boxes), but they're on land so that's not much use.

The pole displacement method is described in sec 5 [here](https://github.com/nikizadehgfdl/grid_generation/blob/dev/ocean_grid_generator_guide.pdf) (possibly out of date). 

The first step starts with a normal lat-lon grid. Could we choose a finer dy poleward of the join in this initial lat-lon grid, so it results in a smaller discontinuity (and squarer cells) in the Ross and Weddell when the pole is displaced? ie choose an initially discontinuous dy which becomes (nearly) continuous in these two regions after stretching? 

If we can compensate for the discontinuity this way, we will be free to move the pole much closer to the join in East Antarctica (dy will be really tiny there, but that doesn't matter as it's on land), and thereby get larger dx at the Ross and Weddell grounding lines.

![Image](https://github.com/user-attachments/assets/ff4f27e7-568f-4592-9f89-36806cf77f87)

**Comment by aekiss** [comment]
Also, we have discontinuous dy at the tripolar join in ACCESS-OM2 and it's apparently never been a problem

![Image](https://github.com/user-attachments/assets/bf61e969-b59e-4e19-9df9-de1d2006c05c)
